editor-components/content/content16/content16.tsx

```diff
@@ -1,129 +1,325 @@
-                      <ComposerLink
-                        key={indexButtons}
-                        path={item.value[1].value}
-                      >
-                        <button
-                          className={`${this.decorateCSS("button")}`}
-                        >
-                          {item.value[0].value}
-                        </button>
-                      </ComposerLink>
-                    );
-                  }
-                )}
+          {(isTitleExist || isSubTitleExist) && (
+            <header>
+              {isTitleExist && (
+                <h1 className={this.decorateCSS("title")}>
+                  {title}
+                </h1>
+              )}
+                <hr className={this.decorateCSS("faint-line")} />
+              {isSubTitleExist && (
+                <h2 className={this.decorateCSS("subtitle")}>
+                  {subtitle}
+                </h2>
+              )}
+            </header>
+          )}
+          <main className={this.decorateCSS("wrapper")}>
+            {items.length > 0 && (
+              <div className={this.decorateCSS("slider-parent")}>
+                <Slider {...settings} className={this.decorateCSS("carousel")}>
+                  {items.map((item: CardType, index: number) => (
+                    <article
+                      className={`${this.decorateCSS("slider-inner-div")} ${this.getComponentState("prevSlide") === index ? this.decorateCSS("prevSlide") : ""} ${this.getComponentState("nextSlide") === index ? this.decorateCSS("nextSlide") : ""}`}
+                      key={`sld-16-${index}`}
+                    >
+                      <div className={this.decorateCSS("content-div")}>
+                        {item.image && (
+                          <div className={this.decorateCSS("img-div")}>
+                            <a href={item.url} target="_blank" rel="noopener noreferrer">
+                              <img
+                                alt={item.imageTitle || item.imageSubtitle}
+                                src={item.image}
+                                className={this.decorateCSS("img")}
+                              />
+                            </a>
+                          </div>
+                        )}
+                        {(item.imageSubtitle || item.imageTitle) && (
+                          <div className={this.decorateCSS("item-page")}>
+                            {item.imageSubtitle && (
+                              <h3 className={this.decorateCSS("first-item")}>
+                                {item.imageSubtitle}
+                              </h3>
+                            )}
+                            {item.imageTitle && (
+                              <h2 className={this.decorateCSS("item-title")}>
+                                {item.imageTitle}
+                              </h2>
+                            )}
+                          </div>
+                        )}
```

Comment:
they should be casted to string while doing empty check, because they return JSX element.